app/models/katello/pool.rb

```diff
@@ -65,7 +65,7 @@ def products
     end
 
     def host
-      Katello::Host::SubscriptionFacet.find_by_uuid(host_id).try(:host) if host_id
+      ::Host.find(hypervisor_id) if hypervisor_id
```

Comment:
Should @jlsherrill's recommendation to use find_by(:id => ID) apply here as well?

Reply:
actually, don't both those methods do the same thing?  Do we need both?

Reply:
I kept it to not break anything, but we really should just use  'hypervisor', let me check where host is used.